src/preprocessing/passes/symmetry_breaker.cpp

```diff
@@ -128,27 +137,40 @@ PreprocessingPassResult SymBreakerPass::applyInternal(
 {
   Trace("sym-break-pass") << "Apply symmetry breaker pass..." << std::endl;
   // detect symmetries
-  std::vector<std::vector<Node>> part;
+  std::vector<std::vector<Node>> sterms;
   symbreak::SymmetryDetect symd;
-  symd.getPartition(part, assertionsToPreprocess->ref());
+  symd.computeTerms(sterms, assertionsToPreprocess->ref());
+  if (!sterms.empty())
+  {
+    Trace("sb-constraint") << "; found symmetry" << std::endl;
+  }
   if (Trace.isOn("sym-break-pass"))
   {
-    Trace("sym-break-pass") << "Detected symmetry partition:" << std::endl;
-    for (const std::vector<Node>& p : part)
+    if (sterms.empty())
```

Comment:
this `if-then-else` can also easily be embedded in the trace below. And you will not get in the `for` anyway when `sterms` is empty